Intersect.Client/Interface/Menu/LoginWindow.cs

```diff
@@ -272,40 +199,37 @@ private void LoadCredentials()
             return;
         }
 
-        mUsernameTextbox.Text = name;
+        _txtUsername.Text = name;
         var pass = Globals.Database.LoadPreference("Password");
         if (string.IsNullOrEmpty(pass))
         {
             return;
         }
 
-        mPasswordTextbox.Text = "****************";
-        mSavedPass = pass;
-        mUseSavedPass = true;
-        mSavePassChk.IsChecked = true;
+        _txtPassword.Text = "****************";
+        _savedPass = pass;
+        _useSavedPass = true;
+        _chkSavePass.IsChecked = true;
     }
 
     private static string ComputePasswordHash(string password)
     {
-        using (var sha = new SHA256Managed())
-        {
-            return BitConverter.ToString(sha.ComputeHash(Encoding.UTF8.GetBytes(password ?? ""))).Replace("-", "");
-        }
+        using var sha = new SHA256Managed();
+        return BitConverter.ToString(sha.ComputeHash(Encoding.UTF8.GetBytes(password ?? ""))).Replace("-", "");
     }
```

Comment:
[Derived cryptographic types are obsolete](https://learn.microsoft.com/en-us/dotnet/fundamentals/syslib-diagnostics/syslib0021)

Try the following (shorter and non obsolete) code instead:

```csharp
    private static string ComputePasswordHash(string password) =>
        BitConverter.ToString(SHA256.HashData(Encoding.UTF8.GetBytes(password ?? string.Empty))).Replace("-", string.Empty);
```